crates/pop-cli/src/commands/install/mod.rs

```diff
@@ -2,32 +2,234 @@
 
 use anyhow::Context;
 use clap::Args;
+use cliclack::{confirm, log, outro};
+use duct::cmd;
+use os_info::Type;
 use tokio::{fs, process::Command};
 
 #[derive(Args)]
 #[command(args_conflicts_with_subcommands = true)]
-/// Setup user environment for substrate development
-/// Runs script `scripts/get_substrate.sh`
-pub(crate) struct InstallArgs;
+/// Setup user environment for development
+pub(crate) struct InstallArgs {
+	/// Before install all the dependencies needed, do not ask the user for confirmation.
```

Comment:
"Automatically install all dependencies required without prompting for confirmation."